string2pdb/stringscore

```diff
@@ -99,20 +157,34 @@ def main():
     # Read tsv data into pandas dataframe:
     interactors_df = pd.read_csv(StringIO(interactors_response.text), sep='\t')
 
-    # Filter interactors:
-    filtered_interactors = interactors_df[interactors_df['score'] >= args.threshold]
+    # Filter interactors with both score >= threshold and dscore > 0:
+    # With database score > 0: No retrieval of unreviewed interactors
+    filtered_interactors = interactors_df[(interactors_df['score'] >= args.threshold) & (interactors_df['dscore'] > args.dthreshold)].copy()
```

Comment:
nitpick, any reason in particular you're `.copy()`ing?

Reply:
yes, i do it to avoid getting the following warning when I try to modify the  filtered_interactors df in line 164: 
```
/data/user/elenikia/PPI2PDB/string2pdb/./stringscore:164: 
SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  filtered_interactors["Interactor_UniProt_AC"] = filtered_interactors["stringId_B"].apply(
```